client/.gitignore

```diff
@@ -22,4 +22,3 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 node
-.env
```

Comment:
ahhh sorry I thought this was you're server `.gitignore` - it looks like the client _does_ have a `.env` file so I guess you do need this line 🙈